eclipse/src/saros/ui/views/SarosView.java

```diff
@@ -661,6 +674,7 @@ private void createActions() {
     registerAction(new SendFileAction());
     registerAction(new ChangeColorAction());
     registerAction(new RemoveUserAction());
+    registerAction(new RequestSessionInviteAction());
```

Comment:
Could you make this feature optional via introducing a properties in `saros.properties`? Otherwise, we would deploy the new button without deploying the server. This would confuse users.

Reply:
We would not deploy this button to any users, because it is only displayed, for servers is in the contact list.